patches/warp-contracts+1.4.39.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/node_modules/warp-contracts/lib/cjs/core/modules/impl/ArweaveGQLTxsFetcher.js b/node_modules/warp-contracts/lib/cjs/core/modules/impl/ArweaveGQLTxsFetcher.js
-index b94141d..0e5e6e4 100644
+index b94141d..fb58060 100644
 --- a/node_modules/warp-contracts/lib/cjs/core/modules/impl/ArweaveGQLTxsFetcher.js
 +++ b/node_modules/warp-contracts/lib/cjs/core/modules/impl/ArweaveGQLTxsFetcher.js
```

Comment:
are we using cjs here? do we need to apply the patch on esm?

Reply:
it's build output of typescript in the tarball, it's the one that ends up being used in the current setup